internal/docstore/mongodocstore/mongo.go

```diff
@@ -146,30 +146,24 @@ type collection struct {
 	idFunc  func(docstore.Document) interface{}
 }
 
-// Options controls collection behaviors.
-type Options struct {
-	// The name of the document field to use for the document ID (MongoDB's _id field).
-	// If neither IDField nor IDFunc is set, then an IDField of "_id" is used.
-	IDField string
-
-	// A function that accepts a document and returns the value to be used for the
-	// document ID (MongoDB's _id field).
-	// IDFunc should return nil if the document is missing the information to
-	// construct an ID. This will cause all actions, even Create, to fail.
-	IDFunc func(docstore.Document) interface{}
+// OpenCollection opens a MongoDB collection for use with Docstore.
+// The idField argument is the name of the document field to use for the document ID
+// (MongoDB's _id field).
```

Comment:
Add a comment that empty `idField` means using default "_id" name.

Reply:
Done.